firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -373,6 +401,23 @@ private boolean isAuthTokenExpired(PersistedFidEntry persistedFidEntry) {
   private long currentTimeInSecs() {
     return TimeUnit.MILLISECONDS.toSeconds(clock.currentTimeMillis());
   }
+
+  private static final class AwaitListener implements OnCompleteListener<Void> {
```

Comment:
you may consider to move this(or some other code too) into a separate class. (The current class is already too big)

Reply:
Done. Thanks.